moveit_msgs/PlanningSceneWorld world
moveit_msgs/RobotState start
string[] group_names
geometry_msgs/Pose[] goals
